### PR TITLE
fix: Mobile updates

### LINKED ISSUE #20: Mobile SCSS
Needs to look more practical on mobile

### PR DESCRIPTION
**Pull request by Sean1572**
closes #20 

This PR features a series of fixes to ensure the website is useable on mobile. By useable I mean:
- Stays within width of phone
- Can be navigable from nav bar
- Main content is readable

This does not take into account making the styling look good. I think that should be a separate PR outside of MVP handled once the site become live.

### CODE REVIEW

**Comment by ntlhui** [comment]
It'd be really cool to be able to swipe on the carousel

**Comment by ntlhui** [comment]
I feel like the standard menu should be the hamburger icon?  not the arrow... but maybe that specifically an Android standard?

**Comment by ntlhui** [comment]
The menu spacing seems uneven

**Review by ntlhui — CHANGES_REQUESTED** [review]
See comments

**Comment by Sean1572** [comment]
> I feel like the standard menu should be the hamburger icon? not the arrow... but maybe that specifically an Android standard?

Technically speaking, the menu isn't a hamburger so an arrow is fine. I've seen arguments that hamburgers are more recognizable as menu icons. Either way I think we will be okay

**Comment by Sean1572** [comment]
> It'd be really cool to be able to swipe on the carousel

good idea

**Comment by Sean1572** [comment]
> 
> 
> The menu spacing seems uneven

quick check, as in the spacing between menu items?

File: assets/c.png
Comment (ntlhui) [inline assets/c.png:1]: What is this for?

**Review by ntlhui — APPROVED** [review]

**Pull request merged**